docs/install-manual.md

```diff
@@ -66,13 +66,17 @@ This example uses the Docker Hub based `token` discovery service. Log into **eac
 
 		$ docker run -d swarm join --addr=172.31.40.100:2375 token://6856663cdefdec325839a4b7e1de38e8
 
-3. Start the Swarm manager on any machine or your laptop. 
+We are done with the per node setup, and will now set up the manager to control the nodes in the swarm.
+
+1. Start the Swarm manager on any machine or your laptop. 
 
 	The following command illustrates how to do this:
 
-		docker run -d -p <swarm_port>:2375 swarm manage token://<cluster_id>
+		docker run -d -p <manager_port>:2375 swarm manage token://<cluster_id>
+
+	The manager will now be exposed and listening on `<manager_port>`.
```

Comment:
Tech docs take place in the now.  The rule of thumb is to avoid future tense. It also, you'll notice makes the sentence more complex.

so...

The manager is exposed and listening on `manager_port`.